pkg/controller/controller.go

```diff
@@ -855,6 +855,7 @@ func (p *csiProvisioner) Delete(volume *v1.PersistentVolume) error {
 				},
 			})
 			if err != nil {
+				klog.Errorf("failed to get secretreference for volume %s: %s", volume.Name, err.Error())
```

Comment:
I mean instead of logging the error and then returning the error (which will get logged again), we should just return an error with this extra msg.

Reply:
yeah, I went through the code, we are not logging and return an error. sorry for the suggestion to add  logging here